phangsPipeline/casaCubeRoutines.py

```diff
@@ -85,33 +177,74 @@ def get_mask(infile, huge_cube_workaround=True):
     Get a mask from a CASA image file. Includes a switch for large cubes, where getchunk can segfault.
     """
 
-    if huge_cube_workaround:
-        os.system('rm -rf ' + infile + '.temp_deg_ordered')
-        casaStuff.imtrans(imagename=infile + '.temp_deg', outfile=infile + '.temp_deg_ordered',
-                          order='0132')
-
-        casaStuff.makemask(mode='copy', inpimage=infile + '.temp_deg_ordered',
-                           inpmask=infile + '.temp_deg_ordered:mask0',
-                           output=infile + '.temp_mask', overwrite=True)
-
-        casaStuff.exportfits(imagename=infile + '.temp_mask',
-                             fitsimage=infile + '.temp.fits',
-                             stokeslast=False, overwrite=True)
-        hdu = pyfits.open(infile + '.temp.fits')[0]
-        mask = hdu.data.T[:, :, 0, :]
-
-        os.system('rm -rf ' + infile + '.temp_deg_ordered')
-        os.system('rm -rf ' + infile + '.temp_mask')
-        os.system('rm -rf ' + infile + '.temp.fits')
-
-    else:
-        myia = au.createCasaTool(casaStuff.iatool)
-        myia.open(infile+'.temp_deg')
-        mask = myia.getchunk(getmask=True)
-        myia.close()
-
-    os.system('rm -rf ' + infile + '.temp_deg')
-
+    #if huge_cube_workaround:
```

Comment:
We should clean up the unused `huge_cube_workaround` code throughout.

Reply:
Would this break Tom's scripts for some earlier data reduction, or other parts of the pipeline if there is a `huge_cube_workaround` leftover? For compatible reason I tend to keep it...